test/manifest-creation.test.ts

```diff
@@ -29,13 +29,13 @@ describe('ManifestCreation', () => {
 
     test('writes new webhook channel to .env', async () => {
       await setup.createWebhookChannel()
-      expect(setup.updateEnv).toHaveBeenCalledWith({"WEBHOOK_PROXY_URL": "https://smee.io/1234abc"})
+      expect(setup.updateEnv).toHaveBeenCalledWith({ 'WEBHOOK_PROXY_URL': 'https://smee.io/1234abc' })
```

Comment:
The primary blocker with this refactor is that `updateEnv()` is a private method in the `ManifestCreation` class.

https://github.com/probot/probot/blob/2ff0ec6b20a9db29d5d39899e1b78110dc6cf6e8/src/manifest-creation.ts#L76-L78

Not sure what the right next step is. Doesn't seem ideal to expose the method in order for testing, but that's an option. What are your thoughts on how to approach this issue?

Reply:
Yea, we did keep that method private since it does allow env changes which could be intrusive. This is the main reason I wrote this test in js to begin with. I'm honestly not sure how to address this. I don't think it's a huge deal if we test if `updateEnv` is called , so much as I wanted to test what it would be called with. 

I'm on the fence about whether to make the method public or just not test it.

Thoughts @bkeepers or @tcbyrd ?

Reply:
If I remember right, we added that method just to make it easier to mock out in tests. I think it's probably fine to make it public.

Reply:
👍